clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1173,6 +1182,9 @@ public void run() {
         pushJobHeartbeatSender.stop();
       }
       inputDataInfoProvider = null;
+      if (pushJobSetting.rmdSchemaDir != null) {
+        HadoopUtils.cleanUpHDFSPath(pushJobSetting.rmdSchemaDir, true);
```

Comment:
If the push job fails to clean up the rmd tmp dir, I assume it will be cleaned up by Grid later automatically, right?

Reply:
I'm not sure about but I think the answer is NO as I've found out people have written bunch of tmp data to `/tmp/`.
We apparently don't want to fail the job due to this but I added one log to track this.